### PR TITLE
Add options for graph refresh

### PR DESCRIPTION
**Pull request by Ladas**
Add options for graph refresh, by default graph refresh is switched off

### CODE REVIEW

**Comment by Ladas** [comment]
@miq-bot assign @durandom

**Comment by Ladas** [comment]
@blomquisg @durandom time has come to switch the graph refresh as default, it's also a nice time to rename the settings, inventory_object_refresh sounds weird

File: spec/models/manageiq/providers/amazon/cloud_manager/refresher_spec.rb [hunk 1]
```diff
@@ -9,6 +9,17 @@
     @ems.update_authentication(:default => {:userid => "0123456789ABCDEFGHIJ", :password => "ABCDEFGHIJKLMNO1234567890abcdefghijklmno"})
   end
 
+  before(:each) do
+    settings                                  = OpenStruct.new
```
Comment (durandom) [inline spec/models/manageiq/providers/amazon/cloud_manager/refresher_spec.rb]: I've been told we should stub settings like this https://github.com/ManageIQ/manageiq-providers-amazon/pull/158/files

Comment (Ladas) [inline spec/models/manageiq/providers/amazon/cloud_manager/refresher_spec.rb]: ok, let's do that once we have the helpers, leaving this way for now

**Comment by Ladas** [comment]
@durandom @blomquisg changing the graph refresh as off by default, but adding the settings, so we can easily turn it on.

Are you fine about the naming? inventory_object_refresh sounds a bit weird

**Pull request merged**